di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/utils/HMRCUserInfoResponse.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.di.ipv.stub.orc.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.JOSEException;
+import org.slf4j.LoggerFactory;
+import uk.gov.di.ipv.stub.orc.models.InheritedIdentityJWT;
+import uk.gov.di.ipv.stub.orc.models.UserInfo;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+public class HMRCUserInfoResponse {
+    private static final org.slf4j.Logger logger =
+            LoggerFactory.getLogger(HMRCUserInfoResponse.class);
+
+    public static String generateResponse(
+            String userId,
+            String vtr,
+            boolean duringMigration,
+            String credentialSubject,
+            String evidence)
+            throws NoSuchAlgorithmException, InvalidKeySpecException, JOSEException {
+        UserInfo userInfo = new UserInfo();
+        userInfo.setInheritedIdentityJWT(
+                duringMigration
+                        ? new InheritedIdentityJWT(
+                                InheritedIdentityJWTBuilder.generate(
+                                                userId, vtr, credentialSubject, evidence)
+                                        .serialize())
+                        : null);
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This objectMapper could be moved to a static field to avoid creating a new one every time this method is called.